src/main/java/thePackmaster/cards/anomalypack/FreeSpace.java

```diff
@@ -0,0 +1,63 @@
+package thePackmaster.cards.anomalypack;
+
+import com.megacrit.cardcrawl.actions.common.DiscardAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.actions.anomalypack.WonderAction;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.util.Wiz.atb;
+
+public class FreeSpace extends AbstractPackmasterCard {
+    public static final String ID = SpireAnniversary5Mod.makeID("FreeSpace");
+    private static final CardStrings cardStrings = CardCrawlGame.languagePack.getCardStrings(ID);
+    public static final String NAME = cardStrings.NAME;
+    public static final String DESCRIPTION = cardStrings.DESCRIPTION;
+    public static final String UPGRADE_DESCRIPTION = cardStrings.UPGRADE_DESCRIPTION;
+    private static final int COST = 1;
+
+    public FreeSpace() {
+        super(ID, COST, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        this.isEthereal = true;
+    }
+
+    @Override
+    public void upp() {
+        this.isEthereal = false;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        Wiz.atb(new DrawCardAction(1));
+    }
+
+    @Override
+    public void triggerOnExhaust() {
+        Wiz.atb(new DrawCardAction(1));
+    }
+
+    @Override
+    public void triggerOnManualDiscard() {
+        Wiz.atb(new DrawCardAction(1));
+    }
+
+
+
+    //Upgraded stats.
+    @Override
```

Comment:
This is not needed and already handled by the abstract. You already added the relevant code to upp()